pbft/pbft_operation.hpp

```diff
@@ -77,6 +77,18 @@ namespace bzn
 
         size_t faulty_nodes_bound() const;
 
+        const std::string& get_preprepare() const { return this->preprepare_message; };
```

Comment:
these should return bzn_envelopes

Reply:
Changed get_prepare to return bzn_envelope and propagated that change.